lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss

```diff
@@ -0,0 +1,58 @@
+.reportback__permalink {
+  .header {
+    > .wrapper {
+      text-align: left;
+    }
+  }
+
+  .container {
+    &.-purple {
+      background: $purple;
+    }
+
+    .-white {
+      background: $white;
+    }
+
+    > .wrapper {
+      @include span(16);
```

Comment:
Hmm. Yeah, I think maybe a `card` pattern might be the way to go here. Let's merge this and potentially refactor later.